pkg/csi/service/fcd/vsphere_helper.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fcd
+
+import (
+	"sort"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
```

Comment:
I like this package, but why break consistency and use logrus instead of glog here?

Reply:
the gocsi package that does the code generation for the skeleton puts that there. i think multiple logging libraries is weird, but if we address that, we should do it in a following PR since the plumbing as existed for some time now.

We have a TODO which changes from glog to klog. We should think about this consolidation then.

Reply:
Ahh I see, thanks. I just opened https://github.com/kubernetes/cloud-provider-vsphere/issues/137 cause I didn't see any open issues for it :)

Reply:
Yeah, this stems more from the idea that CSI plugins are separate from K8s, and the core GoCSI code uses `logrus` (and has since inception). It may be a moot point, but I vastly prefer `logrus` (and structured logging in general` to glog/klog. 🤷‍♂️ 

So it wasn't really about having two (I didn't realize until this PR that *any* of the code pulled into the CSI plugin had glog/klog code), it was that the CSI plugin uses `logrus` because `GoCSI` does. I'd really hate to change that.